base/src/main/java/com/smartdevicelink/managers/BaseSdlManager.java

```diff
@@ -188,52 +188,46 @@ protected void checkLifecycleConfiguration() {
-                            hmiLanguage = actualHMILanguage;
-                            language = actualLanguage;
-
-                            if (lcu.getAppName() != null) {
-                                appName = lcu.getAppName();
-                            }
-
-                            if (lcu.getShortAppName() != null) {
-                                shortAppName = lcu.getShortAppName();
+            if(managerListener != null) {
+                final LifecycleConfigurationUpdate lcu = managerListener.managerShouldUpdateLifecycle(actualLanguage, actualHMILanguage);
+
+                ChangeRegistration changeRegistration;
+                changeRegistration = new ChangeRegistration(actualLanguage, actualHMILanguage);
+
+                if (lcu != null) {
+                    changeRegistration.setAppName(lcu.getAppName());
+                    changeRegistration.setNgnMediaScreenAppName(lcu.getShortAppName());
+                    changeRegistration.setTtsName(lcu.getTtsName());
+                    changeRegistration.setVrSynonyms(lcu.getVoiceRecognitionCommandNames());
+                    changeRegistration.setOnRPCResponseListener(new OnRPCResponseListener() {
+                        @Override
+                        public void onResponse(int correlationId, RPCResponse response) {
+                            if (response.getSuccess()) {
+                                try {
+                                    DebugTool.logInfo(TAG, response.serializeJSON().toString());
+                                } catch (JSONException e) {
+                                    DebugTool.logError(TAG, "Error attempting to serialize ChangeRegistrationResponse", e);
+                                }
+
+                                // go through and change sdlManager properties that were changed via the LCU update
+                                hmiLanguage = actualHMILanguage;
+                                language = actualLanguage;
+
+                                if (lcu.getAppName() != null) {
+                                    appName = lcu.getAppName();
+                                }
+
+                                if (lcu.getShortAppName() != null) {
+                                    shortAppName = lcu.getShortAppName();
+                                }
                             }
-
-                            if (lcu.getTtsName() != null) {
-                                ttsChunks = lcu.getTtsName();
-                            }
-
-                            if (lcu.getVoiceRecognitionCommandNames() != null) {
-                                vrSynonyms = lcu.getVoiceRecognitionCommandNames();
-                            }
-                        } else {
-                            DebugTool.logError(TAG, "Change Registration onError: " + response.getResultCode() + " | Info: " + response.getInfo());
-                            retryChangeRegistration();
```

Comment:
Looks like this code was missed, this should not be removed

Reply:
Good catch